06_neighborhoods/Transportation/transit_trips_cost_city.R

```diff
@@ -0,0 +1,283 @@
+
+
+###################################################################
+
+# Transit Metrics: Transportation Costs & Trips - CITY LEVEL
+# SAME Process (for overall, not subgroup, data):
+# 1. For City: Import and reshape data
+# 2. Crosswalk to the cities (from tracts) & aggregate data
+# 3. QC checks
+# 4. Add Data Quality marker
+# 5. Export final files
+
+###################################################################
+
+# SUMMARY-LEVEL VALUES
+# Raw data pulled from https://htaindex.cnt.org/download/
+# the Housing and Transportation (H+T) Affordability Index from the Center for Neighborhood Technology (CNT)
+
+###################################################################
+###################################################################
+
+# 1. Import all the tract-level files and combine into one mega-file with only the relevant variables and years
+
+# 2015
+# bring in all the downloaded CSVs (state-level tracts) & combine them into one nation-wide file for tracts
+# these files can be found in the Urban Box folder -- replace USERNAME
+tracts15files <- list.files(path="C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/2015_tract",
+                            pattern="*.csv")
+print(tracts15files)
+tractpath15 = file.path("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/2015_tract",tracts15files)
+print(tractpath15)
+transport_tracts_2015 <- do.call("rbind",lapply(tractpath15,FUN=function(files){ read.csv(files)}))
```

Comment:
Can also use tidyverse syntax to read in the files.

`transport_tracts_2015 <- map_df(tractpath15, read_csv)`